examples/tekton/01-clusterrole-privileged.yaml

```diff
@@ -0,0 +1,11 @@
+# When https://issues.redhat.com/browse/STOR-828 is implemented in OpenShift, and the Shared Resource CSI Driver can be
+# deemed safe for restricted users, this ClusterRole and its associated binding can be removed.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: my-shared-resource-privileged-role
+rules:
+  - apiGroups: ["security.openshift.io"]
+    resourceNames: ["privileged"]
```

Comment:
yes, creating a new SCC that adds the `csi` volume type but otherwise minimizes where ever possible the power of the SCC we must define before STOR-828 lands narrows the exposure somewhat, but that said, exposing CSI drivers at all leaves the exposure big enough that it is more or less a drop in the bucket in my opinion

but we can go down the seaprate SCC path in app studio for sure @sbose78 .... again, I tried to keep things a bit simple here

Reply:
that said, I just try a "restricted scc + csi volumes" role and it does work

I'll update the PR in a bit

Reply:
ah, great 😎